src/components/dashboard/Dashboard.js

```diff
@@ -190,7 +191,7 @@ class Dashboard extends Component {
         }
 
         if(categoryName !== catStatesName || users !== catStatesUsernames) {
-            let userIdArr = [JSON.parse(localStorage.getItem('user')).userId];
+            let userIdArr = this.user.userId;
```

Comment:
Make the value an array, similar to how it was before with `[ ]` surrounding the value